src/api/app/views/webui2/webui/groups/show.html.haml

```diff
@@ -20,38 +20,50 @@
 
-    .tab-pane#requests-in{ aria: { controls: 'requests-in' }, role: 'tabpanel' }
-      %h3 Incoming Requests
-      = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_in_table', source_url: group_requests_path(@group) })
+      .tab-pane#requests-in{ aria: { controls: 'requests-in' }, role: 'tabpanel' }
+        %h3 Incoming Requests
+        = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_in_table', source_url: group_requests_path(@group) })
 
-    .tab-pane#all-requests{ aria: { controls: 'all-requests' }, role: 'tabpanel' }
-      %h3 All Requests
-      = render(partial: 'webui2/shared/requests_table', locals: { id: 'all_requests_table', source_url: group_requests_path(@group) })
+      .tab-pane#all-requests{ aria: { controls: 'all-requests' }, role: 'tabpanel' }
+        %h3 All Requests
+        = render(partial: 'webui2/shared/requests_table', locals: { id: 'all_requests_table', source_url: group_requests_path(@group) })
 
-    .tab-pane#group-members{ aria: { controls: 'group-members' }, role: 'tabpanel' }
-      %h3 Group Members
-      .obs-dataTable
-        %table.table.table-striped.table-bordered.w-100#group-members-table
-          %thead
-            %tr
-              %th Login Name
-          %tbody
-            - if @group.users.any?
-              - @group.users.each do |user|
-                %tr
-                  %td= link_to(user, user_show_path(user))
-            - else
+      .tab-pane#group-members{ aria: { controls: 'group-members' }, role: 'tabpanel' }
+        %h3 Group Members
+        .obs-dataTable
+          %table.table.table-striped.table-bordered.table-sm.w-100#group-members-table
+            %thead
               %tr
-                %td This group does not contain users.
-        - if User.current.is_admin? || @group.group_maintainers.where(user: User.current).exists?
-          = link_to group_edit_title_path(title: @group) do
-            %i.fas.fa-edit
-            Edit group members
+                %th Login Name
+                %th
+            %tbody
+              - if @group.users.any?
+                - @group.users.each do |user|
+                  %tr
+                    %td= link_to(user, user_show_path(user))
+                    %td.text-nowrap
+                      = link_to('#', data: { toggle: 'modal', target: "#delete-group-user-#{user.id}-modal" }) do
+                        %i.fas.fa-times-circle.text-danger{ title: "Remove user '#{user}' from '#{@group}' group." }
+                      = render(partial: 'remove_user_modal', locals: { group: @group, user: user })
+              - else
+                %tr
+                  %td This group does not contain users.
+          - if User.current.is_admin? || @group.group_maintainers.where(user: User.current).exists?
+            = link_to('#', data: { toggle: 'modal', target: '#add-user-modal' }, id: 'add-user') do
```

Comment:
We should use the class `nav-link` to put some spacing between the link and the table. We do the same in the other views.

Reply:
Removing the `table-sm` class in the `table` fixed the paddings.